supervisord/Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN mkdir -p ${SUPERVISORD_DIR}/conf ${SUPERVISORD_DIR}/bin
 
 COPY --from=builder /supervisord/bootstrap ${SUPERVISORD_DIR}/bin/
 COPY --from=builder /supervisord/execute.sh ${SUPERVISORD_DIR}/bin/
-COPY --from=builder /supervisord/conf/ ${SUPERVISORD_DIR}/
+COPY --from=builder /supervisord/conf/ ${SUPERVISORD_DIR}/conf/
```

Comment:
Does this behaviour has changed between different docker versions as I can build the image using the command `COPY --from=builder /supervisord/conf/ ${SUPERVISORD_DIR}/` ? @geoand

Reply:
Very interesting :)

The truth is that I am using a very recent docker version (`18.03.1-ce`) which might account for the problem.
In that case I propose we leave this open and see if any other community members will be affected

Reply:
So. I will not merge the PR.

Reply:
Sounds good. Let's just keep it open for the the time being